roxctl/central/debug/download_diagnostics.go

```diff
@@ -41,14 +44,20 @@ func downloadDiagnosticsCommand(cliEnvironment environment.Environment) *cobra.C
 			if urlParams != "" {
 				path = fmt.Sprintf("%s?%s", path, urlParams)
 			}
-			return zipdownload.GetZip(zipdownload.GetZipOptions{
+			err := zipdownload.GetZip(zipdownload.GetZipOptions{
 				Path:       path,
 				Method:     http.MethodGet,
 				Timeout:    flags.Timeout(c),
 				BundleType: "diagnostic",
 				ExpandZip:  false,
 				OutputDir:  outputDir,
 			}, cliEnvironment.Logger())
+			if isTimeoutError(err) {
+				cliEnvironment.Logger().ErrfLn(`
+Timeout is reached while creating diagnostic bundle. Increase 'roxctl' timeout:
+'roxctl central debug download-diagnostics --timeout=400s <other parameters'`)
```

Comment:
What if user already used 400s? Better use flag value times two or more
```suggestion
'roxctl central debug download-diagnostics --timeout=2*timeout from flag <other parameters'`)
```

Reply:
But what if user set some small timeout on the opposite, like `1s`? Then they can go through 7-8 attempts before they get right result.

Maybe we should provide some more detailed guidance on timeout in the warning. For example:
```
Timeout value used was 50 seconds, while default timeout value is 300 seconds. If your timeout value is less than a default value, use default value. If your timeout value is more or equal to default value, increase timeout value twice in size.
```
@dhaus67 thoughts?

Reply:
I think it doesn't hurt to be a bit more descriptive here and mention the default timeout value.